### PR TITLE
doing ui tweaks, cleaning and minor fixes

### PR DESCRIPTION
**Pull request by mordrax**
some final cleaning as listed on the trello card as a cleaning list

### CODE REVIEW

**Comment by JulianLeviston** [comment]
Legendary! 🚢 once debug removed. Really nice. Sorry I didn't realise it wouldn't work properly and it was so much pain.

File: Decoders/ProgressionHistoryV3.elm [hunk 1]
```diff
@@ -130,3 +132,32 @@ progressDecoder =
...
+toDateTime : String -> Result String DateTime
+toDateTime dateTime =
+    let
+        regex =
+            Regex.regex "\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}"
+
+        matches =
+            Regex.find Regex.All regex dateTime
+    in
+        case List.head (Debug.log "matches" matches) of
```
Comment (mordrax) [inline Decoders/ProgressionHistoryV3.elm]: nope

**Comment by mordrax** [comment]
wait.. not finished yet... ops WIP

File: Decoders/ProgressionHistoryV3.elm [hunk 2]
```diff
@@ -150,7 +151,7 @@ toDateTime dateTime =
         matches =
             Regex.find Regex.All regex dateTime
     in
-        case List.head (Debug.log "matches" matches) of
+        case List.head matches of
```
Comment (JulianLeviston) [inline Decoders/ProgressionHistoryV3.elm]: Is `Err` part of the Decoder type?

Comment (JulianLeviston) [inline Decoders/ProgressionHistoryV3.elm]: Lol... wow I forgot that one pretty quick :)

http://package.elm-lang.org/packages/elm-lang/core/latest/Result

**Comment by JulianLeviston** [comment]
Done! 🚢

File: Decoders/ProgressionHistoryV3.elm (hunk 2 above)
Comment (mordrax) [inline Decoders/ProgressionHistoryV3.elm]: DateTime.fromISO.. returns a `Result String DateTime`

Comment (mordrax) [inline Decoders/ProgressionHistoryV3.elm]: and the custom decoder requires the decoding functino to return a Result

**Pull request merged**